### ISSUE TITLE
Upgrade from C119 -> C120 on Win x64 via 1.62.x - Nightly (Checking C120)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#34487]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [ ] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.62.70 Chromium: 120.0.6099.28 (Official Build) nightly (64-bit)
-- | --
Revision | b6614a216032ec09a2e067c13ce2dd34ee90c7e2
OS | Windows 11 Version 22H2 (Build 22621.2715)
```

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build(s):

```
Brave | 1.62.74 Chromium: 120.0.6099.35 (Official Build) nightly (64-bit)
-- | --
Revision | 070cd801c60b65bed66cfed719aeab87384ec9b8
OS | Windows 10 Version 22H2 (Build 19045.3636)
```

<details>
<summary>C119 --> C120 on-demand upgrade check - PASSED</summary>

* installed `1.62.73 Chromium: 119.0.6045.163` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.62.74 Chromium: 120.0.6099.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/a51a5bd9-0ca0-47ad-935a-3c8d5d6b2575) | ![image](https://github.com/brave/brave-browser/assets/2602313/420add70-eccf-4400-9541-b0378d5ad2a2) | ![image](https://github.com/brave/brave-browser/assets/2602313/bd178f0b-75e7-4d1e-9c7f-7efb1650acbf) | ![image](https://github.com/brave/brave-browser/assets/2602313/d884d43b-ed9b-40dd-9601-6d40d57ed21a)

</details>

<details>
<summary>C120 --> C120 on-demand upgrade check - PASSED</summary>

* installed `1.62.70 Chromium: 120.0.6099.28` via `BraveBrowserStandaloneNightlySetup_120_1_62_70.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.62.74 Chromium: 120.0.6099.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/24d16c81-8282-4ac9-81e0-72ce1565a1c5) | ![image](https://github.com/brave/brave-browser/assets/2602313/21504963-f25c-41a9-ac2a-cd251fc18136) | ![image](https://github.com/brave/brave-browser/assets/2602313/6a0d0bae-df17-4209-b890-632cc22ba69e) | ![image](https://github.com/brave/brave-browser/assets/2602313/9ce37ca7-280e-453a-937b-3d8545f24085)

```
[11/24/23 17:39:30.748][BraveUpdate:goopdate][8616:8340][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files (x86)\BraveSoftware\Update\Install\{933582F7-70DA-43B0-8B58-58DE4D2DD047}\brave_installer-delta-x64.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[11/24/23 17:39:30.748][BraveUpdate:goopdate][8616:8340][Running installer][C:\Program Files (x86)\BraveSoftware\Update\Install\{933582F7-70DA-43B0-8B58-58DE4D2DD047}\brave_installer-delta-x64.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[11/24/23 17:39:49.212][BraveUpdate:goopdate][8616:8340][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[11/24/23 17:39:49.212][BraveUpdate:goopdate][8616:8340][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[11/24/23 17:39:49.219][BraveUpdate:goopdate][8616:8340][Started process][10744]
[11/24/23 17:39:49.226][BraveUpdate:goopdate][8616:8340][Started process][11716]
[11/24/23 17:39:49.237][BraveCrashHandler][10744:8396][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[11/24/23 17:39:49.238][BraveCrashHandler][10744:8396][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[11/24/23 17:39:49.238][BraveCrashHandler][10744:8396][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[11/24/23 17:39:49.241][BraveCrashHandler][10744:8396][CrashHandler::RunUntilShutdown]
[11/24/23 17:39:49.415][BraveUpdate:goopdate][3032:9684][DllEntry exit][0x00000000]
```

</details>

<details>
<summary>C119 --> C120 background upgrade check - PASSED</summary>

* installed `1.62.73 Chromium: 119.0.6045.163` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{746A1F48-5919-401B-BBCE-9E09FD905006}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.61.39 Chromium: 119.0.6045.33`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/372dee16-155b-4d2c-8b86-1e63d350e66c)
![image](https://github.com/brave/brave-browser/assets/2602313/20804e3f-db58-46dc-b77a-e55634078fa9)
![image](https://github.com/brave/brave-browser/assets/2602313/90544ec0-fce6-4ec2-a811-02c49b196c7f)

</details>

<details>
<summary>C119 --> C119 background upgrade check - PASSED</summary>

* installed `1.61.36 Chromium: 119.0.6045.33` via `BraveBrowserStandaloneNightlySetup_119_1_61_36.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x64`
* started BraveSoftwareUpdateTaskMachineUA{A1B676DC-A4FA-4B4D-B13F-AFBBDDA6B0E9}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.61.39 Chromium: 119.0.6045.33`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/3456c8dc-0e31-4cb0-880d-a7857d00391c) | ![image](https://github.com/brave/brave-browser/assets/2602313/0b2d5c3b-834a-4ef0-bc6b-f1ab673d7d2e) | ![image](https://github.com/brave/brave-browser/assets/2602313/38ebfe1f-07ae-4bdc-87dd-257943a088e1) | ![image](https://github.com/brave/brave-browser/assets/2602313/1225d91e-84f6-4ae2-883b-7a376f994d33)

</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.62.74 Chromium: 120.0.6099.35 (Official Build) nightly (32-bit)
-- | --
Revision | 070cd801c60b65bed66cfed719aeab87384ec9b8
OS | Windows 10 Version 22H2 (Build 19045.3636)
```

<details>
<summary>C119 --> C120 on-demand upgrade check - PASSED</summary>

* installed `1.62.73 Chromium: 119.0.6045.163` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.62.74 Chromium: 120.0.6099.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/057eb17d-f591-4282-b686-e72190f46b47) | ![image](https://github.com/brave/brave-browser/assets/2602313/717e4573-5bba-474e-8d59-37f4d42b1774) | ![image](https://github.com/brave/brave-browser/assets/2602313/e3ef6c80-1930-4ab6-8ac8-711825ecd5c7) | ![image](https://github.com/brave/brave-browser/assets/2602313/025e840a-a04a-4d07-8c2c-fd65157cae05)
</details>

<details>
<summary>C120 --> C120 on-demand upgrade check - PASSED</summary>

* installed `1.62.70 Chromium: 120.0.6099.28` via `BraveBrowserStandaloneNightlySetup32_120_1_62_70.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.62.74 Chromium: 120.0.6099.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/09a1e822-963a-4cb8-897a-c563f0d2355e) | ![image](https://github.com/brave/brave-browser/assets/2602313/87811377-db43-4616-981c-a17f3f96d0e8) | ![image](https://github.com/brave/brave-browser/assets/2602313/30be1a27-4fe7-4fe4-be03-7e42337d97fa) | ![image](https://github.com/brave/brave-browser/assets/2602313/ab1a9a78-cdef-4eb9-b2d5-01972235d9fb)

```
[11/24/23 18:26:47.127][BraveUpdate:goopdate][12704:2808][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{098F3012-9FC9-4901-AD4C-D7E124BE2A0B}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[11/24/23 18:26:47.127][BraveUpdate:goopdate][12704:2808][Running installer][C:\Program Files\BraveSoftware\Update\Install\{098F3012-9FC9-4901-AD4C-D7E124BE2A0B}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[11/24/23 18:27:06.049][BraveUpdate:goopdate][12704:2808][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[11/24/23 18:27:06.049][BraveUpdate:goopdate][12704:2808][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[11/24/23 18:27:06.060][BraveUpdate:goopdate][12704:2808][Started process][2496]
[11/24/23 18:27:06.080][BraveUpdate:goopdate][12704:6764][Started process][13312]
[11/24/23 18:27:06.082][BraveUpdate:goopdate][4132:8876][DllEntry exit][0x00000000]
```

</details>

<details>
<summary>C119 --> C120 background upgrade check - PASSED</summary>

* installed `1.62.73 Chromium: 119.0.6045.163` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{D6A4C791-4244-47B1-9383-798B4B5C2E23}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.62.74 Chromium: 120.0.6099.35`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/brave/brave-browser/assets/2602313/cce2f819-4093-4746-b818-ddf017b596b8) | ![image](https://github.com/brave/brave-browser/assets/2602313/8c4fabb9-fc10-4c3e-8cf4-ba71fe41ca05) | ![image](https://github.com/brave/brave-browser/assets/2602313/534481a9-dfb0-4f96-8d46-72d0f8fd23fc) | ![image](https://github.com/brave/brave-browser/assets/2602313/c00db64e-8e00-4bce-9bfa-851ad428773d)

</details>

<details>
<summary>C120 --> C120 background upgrade check - PASSED</summary>

* installed `1.62.70 Chromium: 120.0.6099.28` via `BraveBrowserStandaloneNightlySetup32_120_1_62_70.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{F63C19F3-1819-4847-B918-F5A6EEBFCA07}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.62.74 Chromium: 120.0.6099.35`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/d8adc0e3-bcdd-4cbc-8702-590ab76f3ea1) | ![image](https://github.com/brave/brave-browser/assets/2602313/f46a81c4-f9da-48ee-96a0-0570ce577ab9) | ![image](https://github.com/brave/brave-browser/assets/2602313/cf874d71-0993-46fd-a0ea-650bdd50687a) | ![image](https://github.com/brave/brave-browser/assets/2602313/6d1096d4-eff5-40d1-b48f-4ba57ded1af2)

</details>
